notebooks/visualization.ipynb

```diff
@@ -0,0 +1,424 @@
+{
```

Comment:
It would be nice to have the cells executed for this notebook

Reply:
Uploaded a new version with come cells executed. Executing all of them makes the notebook too large to use in the browser.